Library/Homebrew/dev-cmd/linkage.rb

```diff
@@ -9,6 +9,9 @@
 #:
 #:    If `--reverse` is passed, print the dylib followed by the binaries
 #:    which link to it for each library the keg references.
+#:
+#:    If `--rebuild` is passed, flushes sqlite3 db row for 'keg.name' and
```

Comment:
Doesn't matter for now but would be good to just reference the cache rather than an implementation (sqlite)